.github/workflows/ci.yml

```diff
@@ -34,6 +34,11 @@ jobs:
         working-directory: ./rider-fsharp
         run: ./gradlew prepare
 
+      - name: Set up .NET SDK
+        uses: actions/setup-dotnet@v1
```

Comment:
If you use `v2` of this action you can actually point to your `global.json` file to set the version. 
See https://github.com/actions/setup-dotnet `global.json in a subdirectory:`

Reply:
Thanks for the tip, I'll try!